c/models/py_ftrl.cc

```diff
@@ -1200,7 +1202,7 @@ mantissa_nbits : int
     Number of bits from mantissa to be used for hashing floats,
     defaults to `10`.
 
-nepochs : int
+nepochs : float
     Number of training epochs, defaults to `1`.
```

Comment:
Perhaps add an explanation here what does it mean for `nepochs` to be fractional

Reply:
Sure, done.